metron-analytics/metron-profiler/README.md

```diff
@@ -210,78 +293,52 @@ This creates a profile...
  * Adds the `length` field from each message
  * Calculates the average as the result
 
-### Topology Configuration
+### Example 4
 
-The Profiler topology also accepts the following configuration settings.
+It is important to note that the Profiler can persist any serializable Object, not just numeric values.  An alternative to the previous example could take advantage of this.  
```

Comment:
This is a very nice new example, showing how to use the stats functions.  Please add a pointer to documentation for those, so the reader can understand what the result data structure 's' looks like.